### DISCUSSION TITLE
No entrypoint error though the .dll file is getting created

### DISCUSSION THREAD

**Discussion by AshwinSridharan0410** [paketo-buildpacks/feedback#268] [Ideas]
I am trying to build a dotnet azure functions app of .net6.

It is throwing me an error like this :-
no entrypoint [abc-Function.dll] found: stat /workspace/abc-Function.dll: no such file or directory

This is my .csproj file :-
![image](https://github.com/paketo-buildpacks/feedback/assets/113506379/1726d921-ae06-41e1-bb34-07d3b7af41a8)
The command that I used:- (I am building it using the dotnet-core buildpack)
pack build azure-functions --buildpack dotnet-core --path=.
![image](https://github.com/paketo-buildpacks/feedback/assets/113506379/d8d26565-5197-45a3-93f1-06315e30f75d)
![image](https://github.com/paketo-buildpacks/feedback/assets/113506379/b8f57cab-fe90-4ab1-a50b-a2e352a2fc02)
![image](https://github.com/paketo-buildpacks/feedback/assets/113506379/d7e78c85-da9c-4094-8664-9f81038e856b)

These are the logs for dotnet azure functions.

Is paketo buildpacks going to support azure functions by any chance

**Comment by sophiewigmore** [comment]
https://github.com/orgs/paketo-buildpacks/discussions/266
